src/doc/trpl/choosing-guarantees.md

```diff
@@ -0,0 +1,417 @@
+% Wrapper Types
+
+// Alternate title: "Choosing your Guarantees"
+
+A recurring theme in Rust programming is that Rust gives one control over the costs and guarantees
+of their program.
+
+There are various "wrapper type" abstractions in the Rust standard library which embody a multitude
+of tradeoffs between cost, ergonomics, and guarantees. Many let one choose between runtime and
+compile time enforcement. This section will thoroughly explain some of these abstractions.
+
+Before proceeding, it is highly recommended that one reads about [ownership][ownership] and
+[borrowing][borrowing] in Rust.
+
+[ownership]: ownership.html
+[borrowing]: references-and-borrowing.html
+
+
+
+# Basic pointer types
+
+## `Box<T>`
+
+[`Box<T>`][box] is an "owned pointer" or a "box". While it can hand out borrowed references to the
```

Comment:
we shouldn't mention 'owned pointer' at all anymore. I would say "Box<T> is commonly called a 'box'. Boxes provide an easy interface to heap-allocating a single value" or something like that.